development/kops/eks-d.tpl

```diff
@@ -159,7 +159,7 @@ spec:
   iam:
     profile: {{ .nodeInstanceProfileArn }}
   {{- end }}
-  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-{{ .ubuntuRelease }}-{{ .architecture }}-{{ .ubuntuReleaseDate }}
+  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-{{ .architecture }}-server-20221018
```

Comment:
I kinda like the parameterization, should we hold onto it?

Reply:
yes. closing this and reopening wiht all the changes